<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inserir</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Nome</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradaNome"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Média</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="entradaMedia"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnInserir">
           <property name="maximumSize">
            <size>
             <width>377</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Inserir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tabelaDados">
         <property name="maximumSize">
          <size>
           <width>16777191</width>
           <height>16777215</height>
          </size>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Média</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Situação</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnOrdenarNome">
           <property name="text">
            <string>Ordenar por Nome</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOrdenarMedia">
           <property name="text">
            <string>Ordenar por Média</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Média da turma</string>
      </property>
      <widget class="QLabel" name="mediaTurma">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>371</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>381</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Maior média</string>
      </property>
      <widget class="QLabel" name="maiorMedia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Menor média</string>
      </property>
      <widget class="QLabel" name="menorMedia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>381</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>27</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
    <addaction name="actionRemover"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
  </action>
  <action name="actionRemover">
   <property name="text">
    <string>Remover</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>entradaNome</tabstop>
  <tabstop>entradaMedia</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnInserir</tabstop>
  <tabstop>tabelaDados</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
